cmd/agent/install_script.sh

```diff
@@ -142,27 +143,27 @@ elif [ $OS = "Debian" ]; then
     if [ ! -z "$cache_output" ]; then
       $sudo_cmd apt-get install -y dirmngr
     fi
```

Comment:
We may wanna check for the debian version here and disable EBPF when the kernel version is too old (with a warning)